### PR TITLE
add delay to end of player death animation

### PR DESCRIPTION
**Pull request by ronnie-dumesh**
Adds a number of frames to the end of the player death animation before the player is considered "dead" by the game and the "game over" screen pops up

### CODE REVIEW

File: core/src/com/fallenflame/game/PlayerModel.java [hunk 1]
```diff
@@ -231,9 +236,10 @@ protected void setFireBuddyOrigin(float x, float y){
     /**
      * Get the position of the firebuddy on the screen in meters
      */
-    public Vector2 getFireBuddyPosition() { return new
-            Vector2(getPosition().x + (getFireBuddyOriginX() + fireBuddyFilmstrip.getRegionWidth()) / drawScale.x / 2.0f,
-            getPosition().y - (getFireBuddyOriginY() / drawScale.y) + fireBuddyFilmstrip.getRegionHeight() / drawScale.y / 2.0f);}
+    public Vector2 getFireBuddyPosition() { return new Vector2(
```
Comment (lsizemore8) [inline core/src/com/fallenflame/game/PlayerModel.java:239]: Why is this changed here?

File: core/src/com/fallenflame/game/PlayerModel.java [hunk 2]
```diff
@@ -435,10 +439,14 @@ public void update(float dt) {
...
                 walkCool--;
-            } else if (frame == filmstrip.getSize() - 1){
+            } else if (deathDelay <= 0){
                 life = LifeState.DEAD;
+            } else if (frame == filmstrip.getSize() - 1) {
+                deathDelay--;
             }
+
         } else if (isAlive()) {
+            animateFireBuddy(angle100);
```
Comment (lsizemore8) [inline core/src/com/fallenflame/game/PlayerModel.java:449]: Also why is this changed?

**Comment by BryanGraeser** [comment]
Other than Laura's comments, it looks good

File: core/src/com/fallenflame/game/PlayerModel.java (hunk 1 above)
Comment (ronnie-dumesh) [reply core/src/com/fallenflame/game/PlayerModel.java:239]: I tried to factor things out into variables but then realized it would look even worse than it does now. There's no differenec in the code except I just changed the line of the initializer

File: core/src/com/fallenflame/game/PlayerModel.java (hunk 2 above)
Comment (ronnie-dumesh) [reply core/src/com/fallenflame/game/PlayerModel.java:449]: Fire Buddy would "animate" while player was dying. It would essentially just keep shaking side-to-side. Here, it does not move at all while player is dying

**Review by lsizemore8 — APPROVED** [review]
Changes make sense. Looks good.
